k-distribution/src/main/scripts/bin/krun

```diff
@@ -119,8 +118,6 @@ present; see below).
                            stdin/stdout via cells with "stream" attribute.
                            Defaults to enabled when performing concrete
                            execution, and disabled when --search is passed.
-      --no-substitution-filtering  Don't filter conjuncts with anonymous
```

Comment:
Why was this removed?

Reply:
It's not used anywhere as far as I can tell. It's not even passed into any sub-commands as far as I can tell.

Reply:
It was supposed to get passed to kprint. I agree it's not currently. That's a bug we should make an issue for, but I don't think it makes sense to delete it in this pr.